src/components/panel/panel.js

```diff
@@ -332,8 +332,9 @@ function MdPanelService($rootElement, $rootScope, $injector) {
    */
   this._defaultConfigOptions = {
     attachTo: $rootElement,
+    bindToController: true,
     scope: $rootScope.$new(true),
-    bindToController: true
+    transformTemplate: angular.bind(this, this.wrapTemplate_)
```

Comment:
I can't change that one. transformTemplate is something mdCompiler calls once it has the template loaded.